python/lib/modeldata/dmod/test/__init__.py

```diff
@@ -0,0 +1,32 @@
+import git
+
+from pathlib import Path
+from typing import Optional
+
+
+def find_git_root_dir(path: Optional[Path] = None) -> Path:
```

Comment:
Try to find a way to run tests without requiring the source code to be part of an initialized version control repository. While rare, people _do_ in fact download and unpack the source without cloning.